tests/openvino/test_diffusion.py

```diff
@@ -94,6 +94,13 @@ def generate_inputs(self, height=128, width=128, batch_size=1):
 
         return inputs
 
+    def get_auto_cls(self, model_arch):
+        if model_arch == "sana":
+            from diffusers import SanaPipeline
+
+            return SanaPipeline
+        return self.AUTOMODEL_CLASS
```

Comment:
is Sana pipeline not registered as a text to image pipeline ?

Reply:
in the latest released diffuers not yet, see comment above